HW0/P6/P6B.py

```diff
@@ -33,4 +49,5 @@ def burnTime(t):
     plt.xlabel('Wait Time (sec)')
     plt.ylabel('Serial Time (sec) / Parallel Time (sec)')
     plt.title('Speedup versus function time')
+    savefig('P6.png', bbox_inches='tight')
     plt.show()
```

Comment:
Yep, more or less exactly how I did it but I think you did a better job labeling your graph